src/io/flutter/logging/FlutterLogView.java

```diff
@@ -484,9 +498,24 @@ private void computeTextAttributesByLogLevelCache() {
     for (Level level : FlutterLog.Level.values()) {
       try {
         final TextAttributesKey key = LOG_LEVEL_TEXT_ATTRIBUTES_KEY_MAP.get(level);
-        final Color color = globalEditorColorsScheme.getAttributes(key).getForegroundColor();
+        final TextAttributes attributes = globalEditorColorsScheme.getAttributes(key);
+        int fontType = attributes.getFontType();
+        final Color effectColor = attributes.getEffectColor();
+        final Integer textStyle = EFFECT_TYPE_TEXT_STYLE_MAP.get(attributes.getEffectType());
+        // TextStyle can be exist even we unchecked in setting page.
+        // only effectColor null when we uncheck setting effect in setting page.
+        // So, we have to check both effectColor & textStyle not null.
```

Comment:
=> 

So, we have to check that both effectColor & textStyle are not null.

?